shared/message/Geometry.proto

```diff
@@ -24,17 +24,11 @@ package message;
 import "Vector.proto";
 import "Matrix.proto";
 
-/**
- * A normal vector and the distance to the line
- */
```

Comment:
[Hesse normal form](https://en.wikipedia.org/wiki/Hesse_normal_form) representation of a line